components/Tabs/index.js

```diff
@@ -7,3 +7,132 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+
+//Version 6
+//Similar to the first version...not sure if this is correct. I got the topics on the page. The style looks off, but I need to work on the other task. COma back to later, if there's time. 
+axios.get(`https://lambda-times-backend.herokuapp.com/topics`)
+    .then(result =>{
+        // console.log(result);
+        const topics = document.querySelector('.topics');
+        topics.appendChild(TabComponent(result.data));
```

Comment:
what IS result.data? Is it a single item or an array? Is it something you could loop through?